test/clt-tests/mysqldump/versions/script_versions.sh

```diff
@@ -47,6 +47,8 @@ if command -v curl >/dev/null 2>&1; then
         echo "✅ No new versions found after MariaDB $LATEST_MARIADB and MySQL $LATEST_MYSQL"
     else
         echo "❗ Please update the versions array and test new versions!"
+        echo "❗ Also update LATEST_MARIADB and LATEST_MYSQL in script and documentation"
```

Comment:
Saying "update LATEST_MARIADB in documentation" is confusing because there is no "LATEST_MARIADB". It makes sense right now, but over time it will become harder to understand.